portfolio.html

```diff
@@ -25,32 +26,50 @@ <h1>
         </nav>
     </header>
 
-    <div class="Portfolio">
+    <div class="about-me">
+
         <h2>Portfolio</h2>
         <body background="assets/images/gravel.png">
+        
         <hr>
 
```

Comment:
This div is going into the navbar. Try assigining the navbar a higher z index so it will stay on top.